staging/src/k8s.io/cloud-provider/app/testing/testserver.go

```diff
@@ -198,14 +192,13 @@ func StartTestServer(t Logger, customFlags []string) (result TestServer, err err
 }
 
 // StartTestServerOrDie calls StartTestServer t.Fatal if it does not succeed.
```

Comment:
The comment would need to be changed. 

Also this changes behavior from exiting the program the just panicking.